spec/request/in_progress_forms_request_spec.rb

```diff
@@ -55,7 +55,7 @@
         subject
         items = JSON.parse(response.body)['data']
         expect(items.size).to eq(2)
-        expect(items.dig(0, 'attributes', 'form_id')).to be_a(String)
```

Comment:
what do you think about adding a few more specs here that use the olive branch header?

Reply:
added (see also #5630)